--- redland-1.0.9/src/Makefile.am.old	2009-12-03 00:19:25.051648341 +0100
+++ redland-1.0.9/src/Makefile.am	2009-12-03 00:20:01.744463692 +0100
@@ -115,6 +115,13 @@
 
 plugin_LTLIBRARIES = 
 
+if STORAGE_FILE
+plugin_LTLIBRARIES += librdf_storage_file.la
+librdf_storage_file_la_SOURCES = rdf_storage_file.c
+librdf_storage_file_la_LDFLAGS = -module -avoid-version
+librdf_storage_file_la_LIBADD = librdf.la
+endif
+
 if STORAGE_TSTORE
 plugin_LTLIBRARIES += librdf_storage_tstore.la
 librdf_storage_tstore_la_SOURCES = rdf_storage_tstore.c
