--- redland-0.9.12/repat/Makefile.am.orig	2002-09-06 20:00:12.000000000 +0200
+++ redland-0.9.12/repat/Makefile.am	2003-05-22 23:18:13.000000000 +0200
@@ -1,10 +1,8 @@
-noinst_LIBRARIES = librepat.a
 noinst_LTLIBRARIES = librepat.la
 
 CPPFLAGS=@CPPFLAGS@ @LIBRDF_XML_CPPFLAGS@
 CFLAGS=@CFLAGS@ @REPAT_FLAGS@
 
-librepat_a_SOURCES = rdfparse.c rdfparse.h
 librepat_la_SOURCES = rdfparse.c rdfparse.h
 
 noinst_PROGRAMS = rdfdump rdftest
--- redland-0.9.12/raptor/configure.in.orig	2003-02-12 13:38:50.000000000 +0100
+++ redland-0.9.12/raptor/configure.in	2003-05-22 23:21:25.000000000 +0200
@@ -104,14 +104,9 @@
 
 # was AC_REPLACE_FUNCS(strcasecmp) but strcasecmp has two alternatives:
 # the other non-standard function stricmp or a local alternative
-LIBOBJS=
-LTLIBOBJS=
 if test $ac_cv_func_stricmp = no -a $ac_cv_func_stricmp = no; then
-  LIBOBJS="$LIBOBJS strcasecmp.o"
-  LTLIBOBJS="$LTLIBOBJS strcasecmp.lo"
+  AC_LIBOBJ([strcasecmp])
 fi
-AC_SUBST(LIBOBJS)
-AC_SUBST(LTLIBOBJS)
 
 
 AC_MSG_CHECKING(whether need to declare optind)
