--- redland-1.0.5/librdf/Makefile.am.orig	2006-11-01 06:12:21.000000000 +0100
+++ redland-1.0.5/librdf/Makefile.am	2007-02-03 16:30:13.848192112 +0100
@@ -118,7 +118,7 @@
 
 librdf_la_LIBADD = \
 @LTLIBOBJS@ @DIGEST_OBJS@ @HASH_OBJS@ @PARSER_OBJS@ \
-@LIBRDF_INTERNAL_LIBS@
+@LIBRDF_INTERNAL_LIBS@ @LIBRDF_EXTERNAL_LIBS@ $(MEM_LIBS)
 
 librdf_la_DEPENDENCIES = \
 @DIGEST_OBJS@ @HASH_OBJS@ @PARSER_OBJS@ \
@@ -171,7 +171,7 @@
 AM_CPPFLAGS=@LIBRDF_INTERNAL_CPPFLAGS@ @LIBRDF_CPPFLAGS@ @LIBRDF_EXTERNAL_CPPFLAGS@
 AM_CFLAGS=@LIBRDF_INTERNAL_CPPFLAGS@ @LIBRDF_CPPFLAGS@ @LIBRDF_EXTERNAL_CPPFLAGS@ $(MEM)
 STANDARD_CFLAGS=@STANDARD_CFLAGS@ $(MEM)
-AM_LDFLAGS=@LIBRDF_LDFLAGS@ @LIBRDF_EXTERNAL_LIBS@ $(MEM_LIBS)
+AM_LDFLAGS=@LIBRDF_LDFLAGS@
 
 LOCAL_LIB_DIR=lib
 
--- redland-1.0.5/configure.ac.orig	2006-10-23 05:18:47.000000000 +0200
+++ redland-1.0.5/configure.ac	2007-02-03 16:29:30.057696634 +0100
@@ -237,7 +237,7 @@
     fi
 
     # Sigh!  And this still might not be good enough.
-    for bdbc_version in 4.4 4.3 4.2 4.1 4.0 4 3.3 3.2 3.1 3 2; do
+    for bdbc_version in 4.5 4.4 4.3 4.2 4.1 4.0 4 3.3 3.2 3.1 3 2; do
       bdbc_major=`echo $bdbc_version | sed -e 's/^\(.\).*$/\1/'`;
       bdbc_version2=`echo $bdbc_version | sed -e 's/^\(.\).\(.\)$/\1\2/'`
 
@@ -275,7 +275,7 @@
   AC_MSG_RESULT($bdb_include_dir)
 
   if test "$bdb_prefix" != no; then
-    nLDFLAGS=$LDFLAGS
+    nLDFLAGS=
     nCPPFLAGS=
 
     if test "$bdb_lib_dir" != /usr/lib; then
@@ -395,6 +395,7 @@
     if test "$have_libdb" = yes; then
       bdb_available="Version $bdb_version (library $bdb_dbname in $bdb_lib_dir)"
       LIBRDF_LIBS="$LIBRDF_LIBS $nLDFLAGS -l$bdb_dbname"
+      LIBRDF_EXTERNAL_LIBS="$LIBRDF_EXTERNAL_LIBS $nLDFLAGS -l$bdb_dbname"
       LIBRDF_CPPFLAGS="$LIBRDF_CPPFLAGS $nCPPFLAGS"
     fi
 
--- redland-1.0.5/redland-config.in.orig	2006-02-16 08:25:42.000000000 +0100
+++ redland-1.0.5/redland-config.in	2007-02-03 16:29:38.302166460 +0100
@@ -120,7 +120,7 @@
 fi
 
 if test "$echo_libs" = "yes"; then
-  echo -L@libdir@ -lrdf @LIBRDF_LDFLAGS@ @LIBRDF_EXTERNAL_LIBS@
+  echo -L@libdir@ -lrdf @LIBRDF_EXTERNAL_LIBS@
 fi
 
 if test "$echo_libtool_libs" = "yes"; then
