--- redland-0.9.16/raptor/Makefile.am.orig	Fri Jan 30 18:30:30 2004
+++ redland-0.9.16/raptor/Makefile.am	Thu Mar  4 13:13:37 2004
@@ -20,20 +20,14 @@
 
 
 bin_PROGRAMS = rapper
-bin_SCRIPTS = raptor-config
 noinst_SCRIPTS = raptor-src-config
-lib_LTLIBRARIES = libraptor.la
-
-include_HEADERS = raptor.h
 
 noinst_HEADERS = raptor_internal.h win32_config.h
 
-man_MANS = rapper.1 libraptor.3 raptor-config.1
+man_MANS = rapper.1
 
 rapper_SOURCES = rdfdump.c raptor_getopt.h
-rapper_LDADD = libraptor.la @RAPPER_EXTRA_OBJS@
-rapper_DEPENDENCIES = libraptor.la @RAPPER_EXTRA_OBJS@
-EXTRA_rapper_SOURCES = getopt.c
+rapper_LDADD = -lraptor
 
 libraptor_la_SOURCES = raptor_parse.c ntriples_parse.c \
 raptor_uri.c raptor_libxml.c raptor_win32.c raptor_locator.c \
@@ -61,8 +55,6 @@
 
 
 
-SUBDIRS= tests win32 examples
-
 TESTS=raptor_uri_test raptor_namespace_test strcasecmp_test \
 raptor_www_test raptor_set_test raptor_xml_test \
 raptor_sequence_test raptor_stringbuffer_test
@@ -121,7 +113,6 @@
 @MAINT@	-groff -man -Thtml -P-l $< | tidy -asxml -wrap 1000 2>/dev/null | perl $(srcdir)/fix-groff-xhtml $@
 
 pkgconfigdir = $(libdir)/pkgconfig
-pkgconfig_DATA = raptor.pc
 
 # Actually it needs turtle_parser.h but nevermind
 turtle_lexer.c: $(srcdir)/turtle_lexer.l turtle_parser.tab.c
--- redland-0.9.16/raptor/rdfdump.c.orig	Sat Jan 24 18:49:40 2004
+++ redland-0.9.16/raptor/rdfdump.c	Thu Mar  4 13:27:23 2004
@@ -273,9 +273,12 @@
 
       case 'i':
         if(optarg) {
-          if(raptor_syntax_name_check(optarg))
+	  /* until raptor update with new raptor_syntax_name_check() function */
+	  raptor_parser *r = raptor_new_parser(optarg);
+          if(r != NULL) {
+	    raptor_free_parser(r);
             syntax_name=optarg;
-          else {
+	  } else {
             int i;
             
             fprintf(stderr, "%s: invalid argument `%s' for `" HELP_ARG(i, input) "'\n",
