--- redland-1.0.6/redland-config.in.orig	2007-05-02 07:17:27.000000000 +0200
+++ redland-1.0.6/redland-config.in	2007-05-12 00:53:52.531481323 +0200
@@ -133,7 +133,7 @@
 fi
 
 if test "$echo_private_libs" = "yes"; then
-  echo @LIBRDF_LDFLAGS@ @LIBRDF_EXTERNAL_LIBS@
+  echo @LIBRDF_EXTERNAL_LIBS@
 fi
 
 if test "$echo_swig" = "yes"; then
--- redland-1.0.9/src/Makefile.am.orig	2009-12-03 00:36:17.658407858 +0100
+++ redland-1.0.9/src/Makefile.am	2009-12-03 00:37:28.227467072 +0100
@@ -190,7 +190,7 @@
 redland.spec redland.spec.in
 
 librdf_la_LDFLAGS = -version-info @LIBRDF_LIBTOOL_VERSION@ \
-@LIBRDF_LDFLAGS@ @LIBRDF_EXTERNAL_LIBS@ $(MEM_LIBS)
+@LIBRDF_EXTERNAL_LIBS@ $(MEM_LIBS)
 
 pkgdata_DATA=
 if STORAGE_MYSQL
